auctionsystem/client.py

```diff
@@ -184,22 +186,22 @@ def rcv_highest(self, item_num, amount):
             self.bidding_items[item_num]['highest'] = False
 
         # Choose whether or not to bid more on this item
-        if self.gui_cb:
-            self.gui_cb(MESSAGE.HIGHEST, item_num)
+        if self.gui_update_cb:
+            self.gui_update_cb(MESSAGE.HIGHEST, item_num)
 
     def rcv_win(self, item_num, name, ip_addr, port_num, amount):
         # Handle victory
-        if self.gui_cb:
-            self.gui_cb(MESSAGE.WIN, item_num, amount)
+        if self.gui_update_cb:
+            self.gui_update_cb(MESSAGE.WIN, item_num, amount)
         else:
             print("You are the winner of item {}, bought for {}!".format(item_num, amount))
         self.bidding_ended(item_num)
 
     def rcv_bid_over(self, item_num, amount):
         # Item has been sold to another client
         self.bidding_ended(item_num)
-        if self.gui_cb:
-            self.gui_cb(MESSAGE.BID_OVER, item_num, amount)
+        if self.gui_update_cb:
+            self.gui_update_cb(MESSAGE.WIN, item_num, amount)
```

Comment:
Shouldn't this be MESSAGE.BID_OVER?

Reply:
Yes it is...that may have been due to the merge conflict.

Reply:
Fixed